tools/clang/unittests/HLSL/CompilerTest.cpp

```diff
@@ -2322,9 +2322,7 @@ TEST_F(CompilerTest, CodeGenAttributeAtVertex) {
 }
 
 TEST_F(CompilerTest, CodeGenConstantArrayInitializer) {
-  CodeGenTestCheck(L"function_call_initializer.hlsl");
-  CodeGenTestCheck(L"function_call_initializer_uint.hlsl");
-  CodeGenTestCheck(L"function_call_initializer_vec.hlsl");
+  CodeGenTestCheckBatchDir(L"initializer");
```

Comment:
I meant that I moved tests around such that all batch tests are under the "batch" directory so that we don't have to add one function here for each new batch directory anymore. Note that you can still test your single subdirectory by using `hcttest file-check <directory-path>`. The only reason I haven't moved "debug" under "batch" yet is because I have pending code reviews that modify it so it would conflict.